### ISSUE TITLE
Node needs to be Long, can't be string

### ISSUE THREAD

**Issue by rastafrange** [pierrenodet/aruku#1]
Hello, I have identified an issue: I wish nodes could be strings as well and not just Long type, as I have IP addresses for nodes, I have to transform them to Long before. Same if nodes are people's name or whatever.
Thanks

**Comment by pierrenodet** [comment]
Indeed, it's because of this line : https://github.com/pierrenodet/aruku/blob/f4a78351688ce8e91ee070ee8e7430353d8b21ee/modules/core/src/main/scala/partition/LocalGraphPartition.scala#L32
